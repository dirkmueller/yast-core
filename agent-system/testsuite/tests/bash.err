[bash] ShellCommand.cc(shellcommand):78 /bin/cat: tests/missing: No such file or directory
[bash] ShellCommand.cc(shellcommand):78 /bin/cat: tests/missing: No such file or directory
[bash] ShellCommand.cc(shellcommand):78 /bin/cat: tests/missing: No such file or directory
