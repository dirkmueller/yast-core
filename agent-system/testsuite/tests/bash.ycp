{
    return Execute (.bash, "/bin/true");
}

{
    return Execute (.bash, "/bin/false");
}

{
    return Execute (.bash, "/bin/cat tests/data1.read");
}

{
    return Execute (.bash, "/bin/cat tests/data2.read");
}

{
    return Execute (.bash, "/bin/cat tests/missing");
}

{
    return Execute (.bash_output, "/bin/cat tests/data1.read");
}

{
    return Execute (.bash_output, "/bin/cat tests/data2.read");
}

{
    return Execute (.bash_output, "/bin/cat tests/missing");
}

{
    return Execute (.bash_output, "/bin/echo $WHAT ; exit 42", $["WHAT":"life"]);
}

{
    return Execute (.bash_background, "/bin/cat tests/data1.read") == 0;
}

{
    return Execute (.bash_background, "/bin/cat tests/data2.read") == 0;
}

{
    // you don't get a message in the log from this one
    return Execute (.bash_background, "/bin/cat tests/missing") == 0;
}
