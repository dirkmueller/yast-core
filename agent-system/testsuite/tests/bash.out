(0)
(1)
(0)
(0)
(1)
($["exit":0, "stderr":"", "stdout":""])
($["exit":0, "stderr":"", "stdout":"Software is like sex. It's better when it's free.\n"])
($["exit":1, "stderr":"/bin/cat: tests/missing: No such file or directory\n", "stdout":""])
($["exit":42, "stderr":"", "stdout":"life\n"])
(true)
(true)
(true)
